### PR TITLE
Do not show "new domain recipients" alert for newly composed messages

### PR DESCRIPTION
**Pull request by piroor**
# Which issue(s) this PR fixes:

N/A

# What this PR does / why we need it:

The alert about "new domain recipients" is designed for replies, but it appears for newly composed messages.
This change suppresses the alert for newly composed messages.

# How to verify the fixed issue:

## The steps to verify:

1. Start Thunderbird.
2. Go to the addons manager.
3. Install this addon.
4. Go to the options page of this addon.
5. Activate "Confirm when any recipients with domains different from any existing recipients are added" ("返信の宛先に今まで含まれていなかったドメインのアドレスが追加された場合に警告する" in Japanese)
6. Start composition of a reply to an existing message.
7. Add a new recipient which has a domain different from any existing recipient and your email address.
8. Try to send the message.
9. Cancel sending.
10. Start composition of a new message.
11. Add a new recipient which has a domain different from your email address.
12. Try to send the message.

## Expected result:

At the step 8:  a confirmation is shown.
At the step 12: no confirmation is shown.

### CODE REVIEW

File: webextensions/background/background.js [hunk 1]
```diff
@@ -83,7 +86,8 @@ browser.runtime.onMessage.addListener((message, sender) => {
             TYPE_TEMPLATE :
             (types.size > 0) ?
               TYPE_EXISTING_MESSAGE :
-              (signature == blankSignature) ?
+              (signature == blankSignature ||
```
Comment (HashidaTKS) [inline webextensions/background/background.js:89]: このPRとは直接関係ないため、少なくともこのPRでは対応しなくて良いと思うのですが、三項演算子のネストは少し読みにくいのではないかと思いました。

**Review by HashidaTKS — APPROVED** [review]
LGTM!

File: webextensions/background/background.js (hunk 1 above)
Comment (piroor) [reply webextensions/background/background.js]: ご指摘の通りなので、early returnのスタイルに変更しました。

**Comment by HashidaTKS** [comment]
問題ないと思います。

**Comment by piroor** [comment]
Thanks!

**Pull request merged**